create.js

```diff
@@ -0,0 +1,90 @@
+const {MongoClient} = require('mongodb');
+
+async function main(){
+    /**
+     * Connection URI. Update <username>, <password>, and <your-cluster-url> to reflect your cluster.
+     * See http://bit.ly/NodeDocs_lauren for more details
+     */
+    const uri = "mongodb+srv://<username>:<password>@<your-cluster-url>/test?retryWrites=true&w=majority";
+
+    /**
+     * The Mongo Client you will use to interact with your database
+     * See bit.ly/Node_MongoClient for more details
+     */
+    const client = new MongoClient(uri);
```

Comment:
Depending on what version of the node driver is being used here, this line has the potential to log one or both of the following notices:

```
(node:11642) DeprecationWarning: current URL string parser is deprecated, and will be removed in a future version. To use the new parser, pass option { useNewUrlParser: true } to MongoClient.connect.
(node:11642) DeprecationWarning: current Server Discovery and Monitoring engine is deprecated, and will be removed in a future version. To use the new Server Discover and Monitoring engine, pass option { useUnifiedTopology: true } to MongoClient.connect.
```

This is encouraging the user to opt-in to new features that will become the default in later version of the driver. To see a version with no warnings at all, you would need the constructor to look like:

```js
const client = new MongoClient(uri, { useNewUrlParser: true, useUnifiedTopology: true });
```

These flags will eventually go away as we release new versions of the driver, so I'm not sure if you want to include them in the blog post, as it can lead to stale info persisting on the internet.

Reply:
I had those params in initially and @mbroadst suggested I remove them in order to future-proof the post.  The initial post in the series (not yet published) addresses how to get rid of the errors.